From 146cbb2510b174d408ab3e10e2acd03be50e0664 Mon Sep 17 00:00:00 2001
From: Gao Pan <b54642@freescale.com>
Date: Fri, 7 Aug 2015 14:41:00 +0800
Subject: [PATCH 0407/1691] MLK-11354 sim: imx: relapce
 devm_request_and_ioremap with devm_ioremap_resource

commit 146cbb2510b174d408ab3e10e2acd03be50e0664 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

The API devm_request_and_ioremap meets compile error
on branch imx_4.1.y. It is recommend to replace the api
with devm_ioremap_resource.

Signed-off-by: Gao Pan <b54642@freescale.com>
---
 drivers/mxc/sim/imx_sim.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/sim/imx_sim.c b/drivers/mxc/sim/imx_sim.c
index 13c11e8..3fc865b 100755
--- a/drivers/mxc/sim/imx_sim.c
+++ b/drivers/mxc/sim/imx_sim.c
@@ -1769,7 +1769,7 @@ static int sim_probe(struct platform_device *pdev)
 		pr_err("Can't get the MEMORY\n");
 		return -ENOMEM;
 	}
-	sim->ioaddr = devm_request_and_ioremap(&pdev->dev, sim->res);
+	sim->ioaddr = devm_ioremap_resource(&pdev->dev, sim->res);
 	dev_dbg(&pdev->dev, "mapped base address: 0x%08x\n", (u32)sim->ioaddr);
 	if (IS_ERR(sim->ioaddr)) {
 		dev_err(&pdev->dev,
-- 
1.9.1

